services/api/src/api/utils.py

```diff
@@ -11,16 +11,16 @@
 from starlette.responses import JSONResponse, Response
 
 ApiErrorCode = Literal[
-    "MissingRequiredParameter",
-    "InvalidParameter",
-    "ResponseNotReady",
-    "ResponseNotFound",
-    "UnexpectedError",
-    "ExternalUnauthenticatedError",
-    "ExternalAuthenticatedError",
     "AuthCheckHubRequestError",
+    "ExternalAuthenticatedError",
```

Comment:
nit: Maybe take advantage of this PR and also move all API exceptions to a new file exceptions.py as in libcommon?

Reply:
yes, for coherence. Let's do it in another PR.